check.js

```diff
@@ -47,7 +47,21 @@ async function parseTest(files) {
       while (true) {
         pos = source.indexOf(PREFIX, prev + 1)
         if (pos === -1) break
-        let newline = source.indexOf('\n', pos)
+        let newline = source.slice(pos).search(/\r?\n/)
```

Comment:
This way is slow (we need to recreate the string in memory).

There is a better with `RegExp.lastIndex` https://github.com/postcss/postcss/blob/main/lib/tokenize.js#L173-L174

Reply:
That's how it could look like:
```javascript
const regexp = /\r?\n/g;
regexp.lastIndex = pos + 1;
regexp.test(source);
let newline = regexp.lastIndex - 2
```

I also tried to use [this `if-else`](https://github.com/postcss/postcss/blob/main/lib/tokenize.js#L175-L179), but it changes nothing.

If I understand correctly, ` - 2` is used to:
1. Compensate `+1` in `pos + 1`
2. Compensate a newline character's index (`\n`)

The problem is in the 2nd case: it will work correctly either for `\n` or `\r\n`. It seems we need 2 regular expressions: one is for ~~monsters~~ `\r\n`, the other is for ~~humans~~ `\n`. If the first one doesn't help, use the other, like this:
```javascript
const WINDOWS_NEW_LINE = /\r\n/g;
WINDOWS_NEW_LINE.lastIndex = pos + 1;
WINDOWS_NEW_LINE.test(source);

let newline;

if (WINDOWS_NEW_LINE.lastIndex !== 0) {
  newline = WINDOWS_NEW_LINE.lastIndex - 2
} else {
  const UNIX_NEW_LINE = /\n/g;
  UNIX_NEW_LINE.lastIndex = pos + 1;
  UNIX_NEW_LINE.test(source);
  newline = UNIX_NEW_LINE.lastIndex - 1
}
```

Does the speed worth it?

Reply:
Nope, it looks too complicated